test/AbstractFactory/ConfigAbstractFactoryTest.php

```diff
@@ -185,8 +187,8 @@ public function testExceptsWhenConfigNotSet()
     {
         $abstractFactory = new ConfigAbstractFactory();
         $serviceManager = new ServiceManager();
-        self::expectException(ServiceNotCreatedException::class);
-        self::expectExceptionMessage('Cannot find a config array in the container');
+        $this->expectException(ServiceNotCreatedException::class);
+        $this->expectExceptionMessage('Cannot find a config array in the container');
```

Comment:
Arggghhh!

We need to likely decide on which syntax we use, so that we don't have commits like this where we flip between styles.

Prior to the last year or so, we were consistently using instance access; in the past year, particularly with some of the updates to add Psalm support, there have been a number of switches to static access. We need to pick one and be consistent everywhere.

For this patch, I'd suggest leaving these alone, as they add a lot of extra noise to the patch.